packages/package.go

```diff
@@ -82,6 +82,7 @@ type BasePackage struct {
 	Title               *string              `config:"title,omitempty" json:"title,omitempty" yaml:"title,omitempty"`
 	Version             string               `config:"version" json:"version"`
 	Release             string               `config:"release,omitempty" json:"release,omitempty"`
+	Source              *Source              `config:"source,omitempty" json:"source,omitempty"`
```

Comment:
Do you want a `yaml:"source,omitempty"` here too? This struct inconsistently uses `yaml` tags.

Reply:
Yeah, good point, we should take a look to these differences. @mtojek do you know why there are fields with different sets of annotations?